tests/fixtures/mod.rs

```diff
@@ -42,23 +48,42 @@ pub fn tmpdir() -> TempDir {
     let tmpdir = assert_fs::TempDir::new().expect("Couldn't create a temp dir for tests");
     let mut files = FILES.to_vec();
     files.extend_from_slice(HIDDEN_FILES);
-    for file in &files {
+    let (files_hard, files_symlink) = files.split_at(1);
+    println!("path = {:?}", tmpdir.path());
```

Comment:
Is this a debug leftover?

Reply:
By the way, you can use `dbg!()` instead as an easier way to achieve exactly the same print!